spec/requests/api/v0/get_one_user_request_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe 'Get One User' do
+  describe 'sends single user by its valid id' do
+    scenario 'happy path' do
+      user1 = create(:user)
+
+      get "/api/v0/users/#{user1.id}"
+
+      user = JSON.parse(response.body, symbolize_names: true)
+
+      expect(response).to be_successful
+      expect(response.status).to eq(200)
+
+      expect(user[:data]).to be_a(Hash)
+
+      user_data = user[:data]
```

Comment:
Ohhhhh! I like how you set the user[:data] to be its own local variable! I think I might have to refactor and follow this!!!